src/app/assets/stylesheets/foundation-overrides.scss

```diff
@@ -121,7 +121,7 @@ tbody tr:nth-child(even) {
   outline: none;
   @include themify($themes) {
     background-color: themed('modalBackgroundColor');
-    color: themed('modalTextColorPrimary');
```

Comment:
Is `modalTextColorPrimary` used elsewhere? Wondering if `modalTextColorPrimary` itself needs to be changed